src/git_symbol.rs

```diff
@@ -0,0 +1,38 @@
+use crate::flags::{IconTheme as FlagTheme, IconTheme};
+use crate::theme::git_symbol::GitSymbolTheme;
+
+pub struct GitSymbols {
+    theme: GitSymbolTheme,
+    separator: String,
+}
+
+impl GitSymbols {
+    pub fn new(theme: FlagTheme, icon_separator: String) -> GitSymbols {
+        let git_theme = match theme {
+            IconTheme::Unicode => GitSymbolTheme::unicode(),
+            IconTheme::Fancy => GitSymbolTheme::default(),
+        };
+
+        Self {
+            theme: git_theme,
+            separator: icon_separator,
```

Comment:
I would prefer not to add this separator here, it looks better without it:

![image](https://user-images.githubusercontent.com/3764335/230782524-00a21ad3-6f08-415c-bb04-8cf91de0bbe1.png)

Reply:
Done. I also removed the separator option (always no space).